todo_app/tests_e2e/e2e_test.py

```diff
@@ -0,0 +1,73 @@
+import os
+from threading import Thread
+
+import pytest
+import requests
+from dotenv.main import find_dotenv, load_dotenv
+from requests.adapters import ConnectionError
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+from todo_app import app
+from todo_app.items.trello.boards import Boards
+from todo_app.items.trello.transport import TrelloTransport
+
+def wait_for_up():
```

Comment:
Nice. I would hope it doesn't take longer to start up than the selenium driver can handle, but this is a good solution if it does.

Reply:
It did seem to be very slow, but my macbook is somewhat underpowered so perhaps that was the reason?